charts/incubator/frigate/templates/_configmap.tpl

```diff
@@ -470,6 +471,8 @@ logs:
 
 {{- define "frigate.mqtt" -}}
 {{- $mqtt := . }}
+{{- if $mqtt.render_config }}
+enabled: {{ ternary "True" "False" $mqtt.enabled }}
```

Comment:
This allows both excluding it completely from config (using upstream defaults)
and also explicitly set enabled to true or false

Reply:
The problem is that upstream defaults won't allow it to start. The default value for mqtt enabled is True, which means mqtt host is required, but its default is blank. This is why CI is failing.

Put another way, the mqtt block is required for the app to start. It must have either `enabled: False` or `host: xxxx` in it.